src/accelerate/commands/menu/selection_menu.py

```diff
@@ -15,13 +15,22 @@
 """
 Main driver for the selection menu, based on https://github.com/bchao1/bullet
 """
+import builtins
+import pkgutil
 import sys
 
 from . import cursor, input
 from .helpers import Direction, clear_line, forceWrite, linebreak, move_cursor, reset_cursor, writeColor
 from .keymap import KEYMAP
 
 
+in_colab = False
+try:
+    in_colab = pkgutil.find_loader("google.colab") is not None
```

Comment:
Please use [this helper function](https://github.com/huggingface/accelerate/blob/64d7b58c4415a5714c25caa5acb428bec26f94e1/src/accelerate/utils/imports.py#L42) here to be consistent with the rest of the library code.

Reply:
> Please use [this helper function](https://github.com/huggingface/accelerate/blob/64d7b58c4415a5714c25caa5acb428bec26f94e1/src/accelerate/utils/imports.py#L42) here to be consistent with the rest of the library code.

Thanks for your advice. I'm looking into it. 
a) Can I use _is_package_available() outside accelerate/utils/imports.py ? 
```python
import builtins
import sys

from ...utils.imports import _is_package_available
from . import cursor, input
from .helpers import Direction, clear_line, forceWrite, linebreak, move_cursor, reset_cursor, writeColor
from .keymap import KEYMAP


in_colab = _is_package_available("google.colab")
```

b) Or I need create another method in imports.py , e.g. 
```
def is_colab_available():
    return _is_package_available("google.colab")
```
puzzled ...

Reply:
Yes you can do a) :-)